templates/taxi/driver_list.html

```diff
@@ -8,6 +9,12 @@ <h1>
       </a>
     </h1>
 
+    <form action="" method="get" class="form-inline">
+      {{ search_form|crispy }}
+
+      <input style="margin-left: 10px" type="submit" value="🔎" class="btn btn-outline-primary">
+    </form>
+
```

Comment:
No need to duplicate code. Better to create additional template and than include it